starboard/build/config/toolchain_variables.gni

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+is_default_toolchain = current_toolchain == default_toolchain
+is_starboard_toolchain = current_toolchain == starboard_toolchain
+is_cobalt_toolchain = current_toolchain == cobalt_toolchain
+is_host_toolchain = current_toolchain == host_toolchain
+is_starboardized_toolchain =
```

Comment:
A starboardized toolchain is one for which we're building with starboard tools instead of native tools (host toolchain/native toolchain). `is_monolithic_starboardized_toolchain` is a misnomer. I'll add a comment